### PR TITLE
fix docker-cli-to-kubectl.md

### PR DESCRIPTION
**Pull request by cewuandy**
1. The index of docker-cli-to-kubectl.md first item has three dots.
2. All fourth layer Topics are modified to second layer.

### CODE REVIEW

File: content/en/docs/reference/kubectl/docker-cli-to-kubectl.md [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+content_template: templates/concept
 reviewers:
 - bgrant0607
 - brendandburns
 - thockin
 ---
 
-You can use the Kubernetes command line tool kubectl to interact with the api. You can use kubectl if you are familiar with docker-cli. However, there are a few differences in the docker-cli commands and the kubectl commands. Each of the following section details a docker subcommand and explains the kubectl equivalent.
-
 {{< toc >}}
```
Comment (kbhawkey) [inline content/en/docs/reference/kubectl/docker-cli-to-kubectl.md]: Is the toc shortcode needed?

File: content/en/docs/reference/kubectl/docker-cli-to-kubectl.md [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
 
-#### docker run
+{{% capture overview %}}
+You can use the Kubernetes command line tool kubectl to interact with the api. You can use kubectl if you are familiar with docker-cli. However, there are a few differences in the docker-cli commands and the kubectl commands. Each of the following section details a docker subcommand and explains the kubectl equivalent.
+{{% /capture %}}
```
Comment (kbhawkey) [inline content/en/docs/reference/kubectl/docker-cli-to-kubectl.md]: - How about removing, `You can use kubectl if you are familiar with docker-cli`. Possibly these edits as well:

You can use the Kubernetes command line tool `kubectl` to interact with the Docker API. However, there are a few
differences between the `docker` commands and the `kubectl` commands. The following sections show a `docker` sub-command and describe the equivalent `kubectl` command.

File: content/en/docs/reference/kubectl/docker-cli-to-kubectl.md (hunk 1 above)
Comment (Bradamant3) [reply content/en/docs/reference/kubectl/docker-cli-to-kubectl.md]: +1 to removing toc shortcode; should not be needed.

Comment (cewuandy) [reply content/en/docs/reference/kubectl/docker-cli-to-kubectl.md]: I removed it, thanks.

**Comment by steveperry-53** [comment]
/approve
@cewuandy I think the suggestions by @kbhawkey are reasonable. I'll leave it to the two of you to decide on the final text. Then I'll give this an LGTM.

File: content/en/docs/reference/kubectl/docker-cli-to-kubectl.md (hunk 2 above)
Comment (cewuandy) [reply content/en/docs/reference/kubectl/docker-cli-to-kubectl.md]: Your suggestion is good. I think "You can use kubectl if you are familiar with docker-cli" is redundancy, too. 
How about edit to the following:

You can use the Kubernetes command line tool kubectl to interact with the API Server. However, there are a few
differences between the docker commands and the kubectl commands. The following sections show a docker sub-command and describe the equivalent kubectl command.

Comment (kbhawkey) [reply content/en/docs/reference/kubectl/docker-cli-to-kubectl.md]: @cewuandy, I apologize for the confusion. I think you need some statement indicating that users familiar with the docker-cli will find using kubectl straightforward or easy to use. You can decide whether the command line tool names should be highlight.

How about this:
You can use the Kubernetes command line tool `kubectl` to interact with the API Server. Using `kubectl` is straightforward if you are familiar with the Docker command line tool. However, there are a few differences between the `docker` commands and the `kubectl` commands. The following sections show a `docker` sub-command and describe the equivalent `kubectl` command.

Comment (MengZn) [reply content/en/docs/reference/kubectl/docker-cli-to-kubectl.md]: @cewuandy I agree with the proposal of @kbhawkey,thanks for your suggestion

**Pull request merged**